README.md

```diff
@@ -25,10 +25,10 @@ Read things here before you start.
 - [Impossibility Results for Distributed Computing](http://www.morganclaypool.com/doi/abs/10.2200/S00551ED1V01Y201311DCT012) [paywall]
 - [Designing Distributed Systems, Brendan Burns](https://azure.microsoft.com/en-us/resources/designing-distributed-systems/) [Free with registration]
 - [Distributed Systems: Concepts and Design, George Coulouris](https://www.amazon.com/Distributed-Systems-Concepts-Design-5th/dp/0132143011) [Amazon Link]
-- [Akka in Action, Second Edition](https://www.manning.com/books/akka-in-action-second-edition)
+- [Akka in Action, Second Edition](https://www.manning.com/books/akka-in-action-second-edition) 
+- [Edge Computing Technology and Applications](https://www.manning.com/books/edge-computing-technology-and-applications) 
```

Comment:
Website link "https://www.manning.com/books/edge-computing-technology-and-applications" is not found. Could you please have a look and update it?